### DISCUSSION TITLE
Localization description

### DISCUSSION THREAD

**Discussion by rickbenetti** [phosphor-icons/homepage#268] [Ideas]
It's possible to send a localization data about icons at figma?

The description at figma help us a lot to find something, but my team was working on a version with portuguese data and a few suggestions to english keywords, if my team/company could be a pull request with a localization to portuguese and other languages help a lot on maintance and update icons.

**Comment by rektdeckard** [comment]
Hi @rickbenetti, we would gladly accept a Portuguese localization and incorporate it into our tag database. Based on the way we build our DB, this could also translate to localized search on our website too!

Off the bat, I would say that if you are using the Figma library, localization is not going to be possible. The Description field is just text, and merging multiple translations of tags into the same text field is not scalable for multiple localizations, and will lead to false positives and a worse experience for the average user. Until Figma libraries have true support for i18n, we would prefer not to do this. Using the Description field for tagging is already a hack as it is.

But if your team is open to using our [plugin](https://www.figma.com/community/plugin/898620911119764089) instead, we can definitely figure out a way to localize the search feature. A few questions to help me figure out the right approach:

1. Does your team tend to use Portuguese as their preferred language at the OS or application level? (to understand if we can detect it via `navigator.language` in a browser context)
2. Would you find it intuitive that, if your preferred language is Portuguese, the search feature would default to using those translations? Or would you prefer an explicit dropdown or control to set the search language?
3. Would you be willing to provide continued translation support for future versions of Phosphor, adding translations for new icons as we release them?

Currently, the source of truth for naming and tag data is not actually in code -- the code is generated from a Google Table we use internally to track project status. The best way for you to get us this data would be to fill out [this spreadsheet](https://tables.area120.google.com/table/9DItHSjduftcfx_tUACkMp?utm_source=share&utm_medium=referral&utm_campaign=tablelink), which I've already taken the liberty of adding you as a collaborator on.

**Reply by rektdeckard** [reply]
If you're adamant on using the Figma library, unfortunately your best bet is to remix our library file and translate the descriptions yourself 😢. Anything else will not scale beyond 1-2 languages.
